### ISSUE TITLE
Select:  Add keyboard interactions for favorites buttons 

### ISSUE THREAD

**Issue by tlabaj** [patternfly/patternfly-react#4650]
**Describe the issue. What is the expected and unexpected behavior?**The favorites buttons are not accessible by keyboard.
They should follow the same pattern as Application Launcher.

**Comment by mcoker** [comment]
@nicolethoen https://github.com/patternfly/patternfly/pull/3431 has been merged. This adds support for `.pf-m-focus` on the parent `.pf-c-select__menu-wrapper` element, which is the element that has the background color change on hover/focus in this menu. So we'll need to move `.pf-m-focus` from `.pf-c-select__menu-item` to `.pf-c-select__menu-wrapper` in the favorites menu when using the keyboard to navigate up/down.

To be consistent, this also adds the same feature to the app launcher, so we will need to update that as well. Would it make sense for that to be done here, or should I open a separate issue for that?

**Comment by nicolethoen** [comment]
@mcoker I can make it part of this PR
